app/assets/javascripts/admin/order_cycles/order_cycles.js.erb.coffee

```diff
@@ -10,17 +10,19 @@ angular.module('admin.orderCycles', ['ngTagsInput', 'admin.indexUtils', 'admin.e
         # using $parse instead of scope[attrs.datetimepicker] for cases
         # where attrs.datetimepicker is 'foo.bar.lol'
         $(element).datetimepicker
-        	dateFormat: 'yy-mm-dd'
-        	timeFormat: 'HH:mm:ss'
```

Comment:
Seconds in timeFormat were giving some issues, we don't need them as we are selection minutes by quarter of an hour.